docs/source/logging/logging.md

```diff
@@ -1,54 +1,121 @@
-* `console`: show logs on standard output (typically your terminal screen) without any rich formatting
-* `info_file_handler`: write logs of level `INFO` and above to `info.log`
+```{note}
+The name of a logger corresponds to a key in the `loggers` section of the logging configuration file (e.g. `kedro`). See [Python's logging documentation](https://docs.python.org/3/library/logging.html#logger-objects) for more information.
+```
 
-The logging handlers that are actually used by default are `rich` and `info_file_handler`.
+You can take advantage of Rich's [console markup](https://rich.readthedocs.io/en/stable/markup.html) in your logging calls:
 
-The project-side logging configuration also ensures that [logs emitted from your project's logger](#perform-logging-in-your-project) should be shown if they are `INFO` level or above (as opposed to the Python default of `WARNING`).
+```python
+log.error("[bold red blink]Important error message![/]", extra={"markup": True})
+```
 
-We now give some common examples of how you might like to change your project's logging configuration.
+## How to customise Kedro logging
 
 ### Using `KEDRO_LOGGING_CONFIG` environment variable
-
-`KEDRO_LOGGING_CONFIG` is an optional environment variable that you can use to specify the path of your logging configuration file, overriding the default Kedro's `default_logging.yml`.
-
-To use this environment variable, set it to the path of your desired logging configuration file before running any Kedro commands. For example, if you have a logging configuration file located at `/path/to/logging.yml`, you can set `KEDRO_LOGGING_CONFIG` as follows:
+To customise logging, specify the path of your logging configuration file by setting the environment variable `KEDRO_LOGGING_CONFIG` to override the default, which is Kedro's `default_logging.yml`. We recommend you put `logging.yml` inside the `conf` folder. For example, you can set `KEDRO_LOGGING_CONFIG` as follows:
 
 ```bash
-export KEDRO_LOGGING_CONFIG=/path/to/logging.yml
+export KEDRO_LOGGING_CONFIG=<project_root>/conf/logging.yml
 ```
 
 After setting the environment variable, any subsequent Kedro commands will use the logging configuration file at the specified path.
 
 ```{note}
 If the `KEDRO_LOGGING_CONFIG` environment variable is not set, Kedro will default to using the logging configuration file at the project's default location of  Kedro's `default_logging.yml`.
 ```
-### Disable file-based logging
 
-You might sometimes need to disable file-based logging, e.g. if you are running Kedro on a read-only file system such as [Databricks Repos](https://docs.databricks.com/repos/index.html). The simplest way to do this is to delete your `conf/base/logging.yml` file. With no project-side logging configuration specified, Kedro uses the default framework-side logging configuration, which does not include any file-based handlers.
+### Show DEBUG level messages
```

Comment:
```suggestion
### How to show DEBUG level messages
```

Reply:
This is now a subsection of "How to customise..." so makes more sense, since it is implicit that you'll have overridden the default and added `logging.yml`